.nightfalldlp/config.json

```diff
@@ -2,5 +2,9 @@
     "detectors": {
         "CREDIT_CARD_NUMBER": "POSSIBLE",
         "PHONE_NUMBER": "LIKELY"
-    }
+    },
+    "tokenWhitelist": [
```

Comment:
just confirming - is this the same nomenclature the API team uses to refer to whitelists/exclusion rule stuff?